components/CreateVirtualCardsForm.js

```diff
@@ -192,7 +192,7 @@ class CreateVirtualCardsForm extends Component {
   onSubmit(e) {
     e.preventDefault();
     const { values, submitting, deliverType } = this.state;
-    if (!submitting && this.form.current.reportValidity()) {
+    if (!submitting && this.form.current.reportValidity && this.form.current.reportValidity()) {
```

Comment:
We should not prevent submit if the method is not supported b the browser

```suggestion
    if (!submitting && (!this.form.current.reportValidity || this.form.current.reportValidity())) {
```